### DISCUSSION TITLE
v5.0.0-beta6

### DISCUSSION THREAD

**Discussion by michaelliunsky** [michaelliunsky/niRvana-theme#18] [General]
- 添加了主题自动更新功能

<hr /><em>This discussion was created from the release <a href='https://github.com/michaelliunsky/niRvana-theme/releases/tag/v5.0.0-beta6'>v5.0.0-beta6</a>.</em>

**Comment by yonglinzhu** [comment]
装v5.0.0-beta5后
1. 如媒体、页面等的移至回收站操作会跳转到空白页。
2. 安装主题页加载不出来。

**Reply by michaelliunsky** [reply]
您好，请尝试清除浏览器所有缓存。主题更新了fontawesome
如还是不行，我需要以下信息：
1跳转空白网页的链接
2请查看主题页浏览器控制台有无报错。
3你的Wordpress版本，PHP版本号，您的网站链接
谢谢

**Reply by yonglinzhu** [reply]
emmmmm...

步骤：
1. 装到beta6主题，清除浏览器缓存后跳转到登录页，然后发现登录不上，贴的报错信息。
![image](https://github.com/michaelliunsky/niRvana-theme/assets/26948420/943ecbbe-1112-48a5-a91b-7c8e43b34cf7)
  PS：我通过备份的data数据回滚后正常登录，然后又安装beta6，点登出再登录也是上面的报错信息。

2. 空白链接：https://aifeel.top/wp-admin/post.php?post=160&action=untrash&_wpnonce=eca28918a3
  PS: 这是我操作页面菜单，将页面移至回收站后跳转的地址，我看过浏览器控制台，无报错信息。

3. Wordpress 6.1.1 PHP：7.4  /  Wordpress 6.3.0 PHP：8.2

**Reply by michaelliunsky** [reply]
wordpress 6.3.0主题已兼容，请注意PHP版本应为7.4
另：将会在今晚紧急检查是否为主题问题，十分抱歉

**Reply by michaelliunsky** [reply]
注意，我已发现出现问题的原因，将会在下一个版本中修复，抱歉。

**Reply by yonglinzhu** [reply]
确实，我是docker：image: wordpress:6.1.1-php7.4-apache。主题一升上面描述的问题皆毕现。

**Reply by michaelliunsky** [reply]
您好，我已经更新了，您看现在可不可以？

**Reply by yonglinzhu** [reply]
已升到beta7,  自测问题已修复。非常感谢。
